src/content/authors/mark-dinn.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Mark Dinn
+title: Tharun Pasam
 image: /images/authors/mark-dinn.jpg
```

Comment:
Chnage the name to "Survey Sort Team"
Change the image to some random image from upstash